atd-vzv/src/views/summary/CrashesByTimeOfDay.js

```diff
@@ -35,90 +36,54 @@ const CrashesByTimeOfDay = () => {
     if (activeTab !== tab) setActiveTab(tab);
   };
 
+  const hourBlockArray = useMemo(
+    () => [...Array(24).keys()].map((hour) => moment({ hour }).format("hhA")),
+    []
+  );
+
```

Comment:
This is fire 🔥 ! Thanks for sharing, learning something new :)

![2020-07-07_11-26-45](https://user-images.githubusercontent.com/5282430/86812543-bed28c00-c044-11ea-8a50-808b1412ff2c.png)

Reply:
For sure, I learned this from @johnclary and his amazing React spacehip dashboard. 🚀